internal/healthcheck/client.go

```diff
@@ -18,7 +16,7 @@ type Client struct {
 
 func NewClient(client *http.Client) *Client {
 	return &Client{
-		rootURL: fmt.Sprintf("http://localhost:%d", fullnode.HealthCheckPort),
```

Comment:
Removes dependency on pkg `fullnode` from pkg `healthcheck`. Healthcheck pkg should not care about fullnodes. Instead inverts dependency such that `fullnode` cares about `healthcheck` in some situations.